wouso/interface/cpanel/views.py

```diff
@@ -1046,6 +1047,22 @@ def get_success_url(self):
     GroupsAdd.as_view()
 )
 
+@permission_required('config.change_setting')
+def race_toggle_can_play(request):
+    try:
```

Comment:
@catalin-oancea, you should check that the request is ajax. You do not want people to send requests in a different way.